test/sortWith.js

```diff
@@ -0,0 +1,89 @@
+var R = require('..');
+var eq = require('./shared/eq');
+
+
+var albums = [
```

Comment:
i am a fan of test data with some character. adds a bit of pleasure to reading the tests that would otherwise be very dry.

Why not Gould's _Goldberg Variations_ (1955)? 😉